src/configuration.ts

```diff
@@ -44,7 +44,7 @@ export interface IConfigurationFile {
      * Other linter options, currently for testing. Not publicly supported.
      */
     linterOptions?: {
-        typeCheck?: boolean;
+        typeCheck?: boolean,
```

Comment:
nit: would prefer to keep this as a semicolon for consistency with the rest of the interface